### ISSUE TITLE
Cannot access the Wiki

### ISSUE THREAD

**Issue by Grossdm** [biotinker/LibreTasks#232]
Hello,

I tried to review the [Wiki](https://github.com/biotinker/LibreTasks/wiki) but this redirects back to the [LibreTasks repository](https://github.com/biotinker/LibreTasks).

I'm eager to give LibreTasks a try!

**Comment by ghost** [comment]
Click on view more branches, go to wiki branch.
Standard structure of many github projects.

**Comment by Grossdm** [comment]
Thank you for the education.

Does this type of wiki structure limit searching the wiki? (I would guess to the selected branch.)

If so that would appear to be not helpful - but I'm not yet familiar with this structure (schema, perhaps?).

**Comment by pataquets** [comment]
Had the same problem.
Fixing this would help not-so-savvy users, who might be confused and fail to give the app a try.

**Comment by Grossdm** [comment]
@biotinker, perhaps there is a procedure that would migrate the wiki data from your code to the actual repo wiki.

I think that @pataquets and I are really expressing what many other users/potential users of the app are experiencing.
That is, I believe many users would be helped if the wiki could be accessed in the standard manner.

Please give this some consideration.

TIA

**Comment by Venryx** [comment]
Agreed, is confusing. That said, as a developer I knew to check the Issues tab when I hit a problem, so managed to find this, but many users wouldn't think (or care enough) to do this.
